code/modules/mob/living/carbon/human/species_types/ethereal.dm

```diff
@@ -126,141 +218,141 @@
 	)
 	return result
 
-/datum/species/elzuose/proc/set_ethereal_light(mob/living/carbon/human/H, current_color)
+/datum/species/elzuose/proc/set_ethereal_light(mob/living/carbon/human/_human, current_color)
 	if(!ethereal_light)
 		return
 
-	var/health_percent = max(H.health, 0) / 100
+	var/health_percent = max(_human.health, 0) / 100
 
 	var/light_range = 1 + (1 * health_percent)
 	var/light_power = 1 + round(0.5 * health_percent)
 
 	ethereal_light.set_light_range_power_color(light_range, light_power, current_color)
 
-/datum/species/elzuose/proc/on_emp_act(mob/living/carbon/human/H, severity)
+/datum/species/elzuose/proc/on_emp_act(mob/living/carbon/human/_human, severity)
 	EMPeffect = TRUE
-	spec_updatehealth(H)
-	to_chat(H, "<span class='notice'>You feel the light of your body leave you.</span>")
+	spec_updatehealth(_human)
+	to_chat(_human, span_notice("You feel the light of your body leave you."))
 	switch(severity)
 		if(EMP_LIGHT)
-			addtimer(CALLBACK(src, PROC_REF(stop_emp), H), 10 SECONDS, TIMER_UNIQUE|TIMER_OVERRIDE) //We're out for 10 seconds
+			addtimer(CALLBACK(src, PROC_REF(stop_emp), _human), 10 SECONDS, TIMER_UNIQUE|TIMER_OVERRIDE) //We're out for 10 seconds
 		if(EMP_HEAVY)
-			addtimer(CALLBACK(src, PROC_REF(stop_emp), H), 20 SECONDS, TIMER_UNIQUE|TIMER_OVERRIDE) //We're out for 20 seconds
+			addtimer(CALLBACK(src, PROC_REF(stop_emp), _human), 20 SECONDS, TIMER_UNIQUE|TIMER_OVERRIDE) //We're out for 20 seconds
 
-/datum/species/elzuose/proc/on_emag_act(mob/living/carbon/human/H, mob/user)
+/datum/species/elzuose/proc/on_emag_act(mob/living/carbon/human/_human, mob/user)
```

Comment:
It's a little unrelated but can you just get rid of the multitool/emag interactions? I don't feel like it fits Elzuose. fine if not

Reply:
Yeah, sure. I'll do it in a separate PR if that's alright.

Reply:
yeah